cli/internal/cmd/iam.go

```diff
@@ -40,3 +41,18 @@ func newIAMCreateCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewIAMDestroyCmd returns a new cobra.Command for the iam destroy subcommand.
+func newIAMDestroyCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "destroy",
+		Short: "Destroy an IAM user and delete local terraform files for the IAM user",
+		Long:  "Destroy an IAM user and delete local terraform files for the IAM user",
```

Comment:
```suggestion
		Long:  "Destroy an IAM user and delete local terraform files for the IAM user.",
```
make sure to be consistent with https://github.com/edgelesssys/constellation/blob/main/dev-docs/conventions.md#cli-reference